src/auxinfo/participant.rs

```diff
@@ -637,14 +675,18 @@ mod tests {
     }
     #[test]
     fn test_run_auxinfo_protocol() -> Result<()> {
+        let QUORUM_SIZE = 3;
         let mut rng = crate::utils::get_test_rng();
-        let mut quorum = AuxInfoParticipant::new_quorum(3, &mut rng)?;
+        let mut quorum = AuxInfoParticipant::new_quorum(QUORUM_SIZE, &mut rng)?;
         let mut inboxes = HashMap::new();
         let mut main_storages: Vec<Storage> = vec![];
         for participant in &quorum {
             let _ = inboxes.insert(participant.id, vec![]);
             main_storages.append(&mut vec![Storage::new()]);
         }
+        let mut outputs = std::iter::repeat_with(|| None)
```

Comment:
This is fancy :fire-emoji:

Reply:
I really like initializing vectors like this, although I actually tried `vec![None; QUORUM_SIZE]` and it wouldn't let me because the `Some` type isn't `Clone` :|

Reply:
:headstone: